### PR TITLE
[release-calendar] Create ChannelTable skeleton

### PR DESCRIPTION
**Pull request by ajwhatson**
This is the basic react components and scss for the ChannelTable. (excluded to instead be added in a different pr is event handling)
Included is:
- the new `ChannelTable.tsx` which loops through an array of objects called `rows` that holds the static information used to create a row for each channel. Each row contains a checkbox square, the channel name, and a button for releases.
- the stylesheets `ChannelTable` and `App`
- mixins within stylesheets, especially for the checkbox square
- small changes to `Calendar` to help with adapting to different screen sizes

With styling, the table will look like this:
Screen Shot 2020-04-15 at 10 48 05 AM

### CODE REVIEW

File: release-calendar/src/types.ts [hunk 1]
```diff
@@ -26,6 +26,16 @@ export enum Channel {
   CREATED = 'created',
 }
 
+export const channels: Channel[] = [
```
Comment (estherkim) [inline release-calendar/src/types.ts:29]: Having this and the `Channel` enum is confusing. Does this belong here or can it go to the `ChannelTable` component?

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * limitations under the License.
+ */
+import * as React from 'react';
+import {channels} from '../../types';
+import {Square} from './Square';
+
+export class ChannelTable extends React.Component<{}, {}> {
+  //TODO(ajwhatson):
+  // add event handling with onClick functions
+  // send state from app carrying Map<Channel, boolean> to map selected channels
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:23]: Might be cleaner to send an array of selected channels instead of a map where the value is a boolean.

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        <div className='title-bar'>
+          <h1>Current Releases</h1>
+        </div>
+        <div className='row-container'>
+          {channels.map((channel) => {
+            const rtv = '1111111111111';
+            return (
+              <div key={channel}>
+                <div>
+                  <button className='row' key={channel} value={channel}>
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:40]: What is the point of 'key' and 'value' here?

File: release-calendar/src/client/components/Square.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+
+export interface SquareProps {
+  channel: string;
+}
+export class Square extends React.Component<SquareProps, {}> {
```
Comment (estherkim) [inline release-calendar/src/client/components/Square.tsx:22]: > the new Square.tsx which is really only separated from the channelTable so that it's styling can also be separated

It's not necessary for this to be a separate component just for styling. CSS supports this just fine.

Comment (ajwhatson) [reply release-calendar/src/client/components/Square.tsx:23]: Before, I had been using a ts function that adds styling when a row is selected. It was pretty patchy so I got rid of the regular buttons and replaced it with checkboxes. Now when a row is selected, the row will change appearance only though css.

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+import '../stylesheets/channelTable.scss';
+import '../stylesheets/square.scss';
+import * as React from 'react';
+import {Channel} from '../../types';
+
+export class ChannelTable extends React.Component<{}, {}> {
+  //TODO(ajwhatson):
+  // add event handling with onClick functions
+  // send state from app carrying array of booleans to represent selected channels
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:25]: array of channels, not booleans.

File: release-calendar/src/client/components/Calendar.tsx [hunk 6]
```diff
@@ -29,6 +30,9 @@ export class Calendar extends React.Component<CalendarProps, {}> {
   render(): JSX.Element {
     return (
       <div className='calendar'>
+        <div className='title-bar'>
+          <h1>Calendar Title</h1>
+        </div>
```
Comment (estherkim) [inline release-calendar/src/client/components/Calendar.tsx:35]: Why is this added?

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+        <div className='row-container'>
+          {this.channels.map((channel) => {
+            const rtv = '1111111111111';
+            return (
+              <div key={channel}>
+                <div className='row'>
+                  <label htmlFor={channel}>
+                    <div className='label'>{channel}</div>
+                    <div className={'square-' + channel}>
+                      <input type='checkbox' id={channel}></input>
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:53]: Can we get the human-readable names? i.e. 'Long Term Stable' instead of 'lts'

File: release-calendar/src/client/stylesheets/square.scss [hunk 8]
```diff
@@ -0,0 +1,41 @@
+.square-lts {
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/square.scss:1]: There's a lot of repetition here. You can do something like `.lts > .square` to specify the background color, or leverage SCSS mixins using the channel as a param - https://sass-lang.com/documentation/at-rules/mixin#arguments

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 9]
```diff
@@ -0,0 +1,91 @@
+
+button:focus {
+    outline: none !important;
+}
+
+body {
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:6]: Move `body` styling to the top level stylesheet.

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../stylesheets/channelTable.scss';
+import '../stylesheets/square.scss';
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:18]: Can you combine the 2 stylesheets? Would be nice to have a 1:1 component to stylesheet ratio.

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+   width: 3em;
+   height: 3em;
+   opacity: 100%;
+   background-color: black;
+}
+  
+/* On mouse-over*/
+.row:hover input ~ .square {
+    opacity: 50% !important;
+}
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:76]: What's going on here? I don't see any changes on hover.

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+  
+/* When checked */
+.row input:checked ~ .square {
+    opacity: 50% !important;
+}
+
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:82]: Also here, I don't see any changed when a row is selected.

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+    padding-top: 0px;
+    width:65%;
+    height: 3em;
+    cursor: pointer;
+    user-select: none;
+    -webkit-appearance: button;
+    box-sizing: border-box;
+    border-width: 1px;
+    border-style: solid;
+    border-color: rgb(216, 216, 216) rgb(209, 209, 209) rgb(186, 186, 186);
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:52]: Turn this into a variable

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 14]
```diff
@@ -0,0 +1,91 @@
+
+button:focus {
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:2]: Probably not best to apply styles to all buttons, but use a class or id selector instead.

File: release-calendar/src/client/components/App.tsx [hunk 15]
```diff
@@ -43,10 +46,18 @@ export class App extends React.Component<{}, AppState> {
...
-      <div>
+      <div className='AMP-Calendar'>
         <Header title='AMP Release Calendar' />
         <div>
-          <Calendar events={this.state.events} />
+          <div className='main-container'>
+            <div className='col-sidePannel'>
+              <ChannelTable />
+            </div>
+            <div className='col-break'></div>
```
Comment (estherkim) [inline release-calendar/src/client/components/App.tsx:56]: Instead of laying an element between the table and calendar, using padding.

File: release-calendar/src/client/components/App.tsx [hunk 16]
```diff
@@ -43,10 +46,18 @@ export class App extends React.Component<{}, AppState> {
 
   render(): JSX.Element {
     return (
-      <div>
+      <div className='AMP-Calendar'>
```
Comment (estherkim) [inline release-calendar/src/client/components/App.tsx:49]: CSS class names are usually kebab style. This would be `amp-calendar`

File: release-calendar/src/client/components/App.tsx [hunk 17]
```diff
@@ -43,10 +46,18 @@ export class App extends React.Component<{}, AppState> {
...
         <Header title='AMP Release Calendar' />
         <div>
-          <Calendar events={this.state.events} />
+          <div className='main-container'>
+            <div className='col-sidePannel'>
```
Comment (estherkim) [inline release-calendar/src/client/components/App.tsx:53]: `col-side-panel`

File: release-calendar/src/client/stylesheets/app.scss [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+    box-sizing: border-box;
+    width: 100%;
+    flex-direction: row;
+    align-items: center;
+    white-space: nowrap;
+    height: 64px;
+}
+
+.main-container{
+    z-index: 1;
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/app.scss:27]: What is this being used for?

File: release-calendar/src/client/stylesheets/app.scss [hunk 19]
```diff
@@ -0,0 +1,86 @@
+.AMP-Calendar{
+    z-index: 0;
+    width: 100%;
+    top:0px;
+    left: 0px;
+    position:fixed;
+}
+
+.header{
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/app.scss:9]: This is unused.

File: release-calendar/src/client/components/App.tsx [hunk 20]
```diff
@@ -43,10 +46,18 @@ export class App extends React.Component<{}, AppState> {
 
   render(): JSX.Element {
     return (
-      <div>
+      <div className='AMP-Calendar'>
         <Header title='AMP Release Calendar' />
         <div>
```
Comment (estherkim) [inline release-calendar/src/client/components/App.tsx:51]: Remove this extra `<div>`

File: release-calendar/src/client/components/Calendar.tsx (hunk 6 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/Calendar.tsx:35]: It helps to horizontally align everything. "Calendar Title" is just a placeholder. I can add a comment about that to clarify.

File: release-calendar/src/client/components/App.tsx (hunk 17 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/App.tsx]: silly me

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 9 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/channelTable.scss]: I moved it to `index.html`. Is that where you meant?

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 11 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/channelTable.scss]: Problems were because I had changed that name for the classNames from the generic .square to more specific .square-lts. I changed this area pretty dramatically with mixins, should be better now.

File: release-calendar/src/client/stylesheets/square.scss (hunk 8 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/square.scss]: oh, cool. I didnt realize mixins could take in arguments. Thank you!

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 14 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/channelTable.scss]: I'm very sure I wont want the outline on any buttons. When the row label used to be a button, I had `outline: none !important;`s everywhere. From what I can tell on the internet, most people do remove it. It's there to make sites more accessible. As long as I check color contrast and html check, hover, before, and after, I should be okay. That said, I can still move it out :)

File: release-calendar/src/client/components/Calendar.tsx (hunk 6 above)
Comment (estherkim) [reply release-calendar/src/client/components/Calendar.tsx:37]: The calendar is pretty self explanatory though. Maybe you can play around with the channel table title.

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 9 above)
Comment (estherkim) [reply release-calendar/src/client/stylesheets/channelTable.scss]: It could work there but I meant `app.scss`. Having styles in both html and scss files would be difficult to maintain.

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 14 above)
Comment (estherkim) [reply release-calendar/src/client/stylesheets/channelTable.scss]: Ok, if you want that to be a site-wide style, move it to the top level stylesheet.

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 21]
```diff
@@ -0,0 +1,66 @@
...
+        <div className='title-bar'>
+          <h1>Current Releases</h1>
+        </div>
+        <div className='row-container'>
+          {this.channels.map((channel) => {
+            const rtv = '1111111111111';
+            return (
+              <div key={channel}>
+                <label className='row' htmlFor={channel}>
+                  <div className={'square-' + channel}>
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:49]: You already have a parent div that has `key={channel}`. Instead of creating a separate `square-{channel}` class, you can create a CSS selector to be .lts > .square

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
+                    <input
+                      type='checkbox'
+                      className='square'
+                      id={channel}></input>
+                    <i></i>
+                  </div>
+                  <div className='label'>{ChannelTitles.get(channel)}</div>
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:56]: This div has a class name `label` inside a `<label>` element which is confusing.

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 23]
```diff
@@ -0,0 +1,118 @@
...
+	}
+}
+
+label {
+  .square {
+    @include square();
+  }
+}
+
+//Colors: (to Esther: should I do label the colors this way or is it unnecessary)
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:76]: Yes, this is fine.

Side note - you can add comments to your own PR instead of writing notes in the code itself.

File: release-calendar/src/types.ts [hunk 24]
```diff
@@ -26,6 +26,20 @@ export enum Channel {
   CREATED = 'created',
 }
 
+interface Inames {
+  [key: string]: string;
+}
+
+export const channelNames: Inames = {
+  'lts': 'Long Term Stable',
```
Comment (estherkim) [inline release-calendar/src/types.ts:34]: Use enum values instead

File: release-calendar/src/types.ts [hunk 25]
```diff
@@ -26,6 +26,20 @@ export enum Channel {
   CREATED = 'created',
 }
 
+interface Inames {
+  [key: string]: string;
+}
+
+export const channelNames: Inames = {
```
Comment (estherkim) [inline release-calendar/src/types.ts:33]: `Inames` look a little weird. Is that what typescript recommends?

Comment (estherkim) [inline release-calendar/src/types.ts:33]: I'm generally ok with keeping this in `types` but this also works somewhere in the client side, since the server won't need this.

Why not combine this with `ChannelTable.channels`?

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 26]
```diff
@@ -0,0 +1,124 @@
...
+    background-color: #{$channel};
+}
+
+.row-button > .lts{
+    @include square($lts);
+}
+
+.row-button > .stable {
+    @include square($stable);
+}
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/channelTable.scss:104]: nit: you can nest classes in CSS like this
```
.row-button {
  .lts {}
  .stable {}
}
```

File: release-calendar/src/client/index.html [hunk 27]
```diff
@@ -6,6 +6,25 @@
   <title>AMP Release Calendar</title>
 </head>
 
+<style>
+body {
```
Comment (estherkim) [inline release-calendar/src/client/index.html:10]: Did you forget to move styles out of this file?

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 13 above)
Comment (estherkim) [reply release-calendar/src/client/stylesheets/channelTable.scss:37]: Was this done?

File: release-calendar/src/client/stylesheets/app.scss [hunk 28]
```diff
@@ -0,0 +1,44 @@
...
+        rgba(0, 0, 0, 0.14) 0px 4px 5px 0px,
+        rgba(0, 0, 0, 0.12) 0px 1px 10px 0px;
+    transition: box-shadow 280ms cubic-bezier(0.4, 0, 0.2, 1);
+    display: block;
+    position: relative;
+    padding: 16px;
+    border-radius: 4px;
+}
+
+.col-side-pannel {
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/app.scss:18]: typo: panel not pannel

File: release-calendar/src/types.ts (hunk 24 above)
Comment (ajwhatson) [reply release-calendar/src/types.ts:34]: When I try to make it the enum, it give me this error. I dont think I'm allowed to.
An index signature parameter type cannot be a union type. Consider using a mapped object type instead.ts

File: release-calendar/src/types.ts (hunk 25 above)
Comment (ajwhatson) [reply release-calendar/src/types.ts:33]: Typescript wants the first letters of words to be capitalized. I'll switch it to `Names` . It seems like people on forums online are very opinionated about it. :)

File: release-calendar/src/client/stylesheets/app.scss (hunk 28 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/app.scss:18]: oop

Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/app.scss:18]: I've totally read that like a million times without noticing. yeet

File: release-calendar/src/client/index.html (hunk 27 above)
Comment (ajwhatson) [reply release-calendar/src/client/index.html:10]: Oh wait totally forgot to say. App.scss isn't imported into every tsx file and therefore it i make changes to body within app.sccs then I wont be changing the body and buttons styling everywhere i want it to be changed.

File: release-calendar/src/client/stylesheets/channelTable.scss (hunk 26 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/channelTable.scss:104]: so much better

File: release-calendar/src/client/stylesheets/app.scss [hunk 29]
```diff
@@ -0,0 +1,64 @@
...
+    .col-side-pannel {
+        width: 70%;
+    }
+}
+
+body {
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
+    font-size: 14px;
+    line-height: 3em;
+    color: #24292e;
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/app.scss:51]: What's this used for?

File: release-calendar/src/client/stylesheets/app.scss [hunk 30]
```diff
@@ -0,0 +1,64 @@
...
+    margin: 0px;
+}
+
+button {
+    -webkit-appearance: none;
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
+    font-size: 14px;
+    line-height: 3em;
```
Comment (estherkim) [inline release-calendar/src/client/stylesheets/app.scss:59]: This is the same as body styles. Could be removed.

File: release-calendar/src/client/components/App.tsx [hunk 31]
```diff
@@ -43,12 +46,17 @@ export class App extends React.Component<{}, AppState> {
 
   render(): JSX.Element {
     return (
-      <div>
+      <>
```
Comment (estherkim) [inline release-calendar/src/client/components/App.tsx:49]: ?

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 32]
```diff
@@ -0,0 +1,64 @@
...
+import * as React from 'react';
+import {Channel} from '../../types';
+
+export class ChannelTable extends React.Component<{}, {}> {
+  //TODO(ajwhatson):
+  // add event handling with onClick functions
+  // send state from app carrying array of selected channels
+  // add app call for most recent releases in each channel
+
+  rowsInfomation = [
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:27]: typo. also you can jut call it `rows`

File: release-calendar/src/client/components/App.tsx (hunk 31 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/App.tsx:49]: It's a React specific thing. https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

File: release-calendar/src/client/components/ChannelTable.tsx (hunk 32 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/ChannelTable.tsx:27]: hehe, 🙃Thanks!

File: release-calendar/src/client/stylesheets/app.scss (hunk 30 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/app.scss:59]: Buttons dont inherit by default. I did change things for readability to
`font-family: inherit;
    font-size: inherit;`

File: release-calendar/src/client/components/ChannelTable.tsx [hunk 33]
```diff
@@ -0,0 +1,65 @@
...
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../stylesheets/channelTable.scss';
+import * as React from 'react';
+import {Channel} from '../../types';
+import {Fragment} from 'react';
```
Comment (estherkim) [inline release-calendar/src/client/components/ChannelTable.tsx:20]: Isn't this already imported because you have `import * as React from 'react'`?

File: release-calendar/src/client/components/App.tsx (hunk 31 above)
Comment (estherkim) [reply release-calendar/src/client/components/App.tsx:49]: I see. Can you make it consistent? Either use `<>` or `<Fragment>` but not both.

File: release-calendar/src/client/components/ChannelTable.tsx (hunk 33 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/ChannelTable.tsx:20]: I just have to write React.Fragment. I thought it was cleaner to just have Fragment, but if it's odd, I can do React.Fragment

File: release-calendar/src/client/components/App.tsx (hunk 31 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/App.tsx:49]: <> doesn't let you add keys which is needed for this line`<Fragment key={row.channel}>` so they are a little different. Should I write React.Fragment everywhere then?

File: release-calendar/src/client/components/ChannelTable.tsx (hunk 33 above)
Comment (estherkim) [reply release-calendar/src/client/components/ChannelTable.tsx:20]: Between importing a module twice and using a namespace, I'd use the namespace.

File: release-calendar/src/client/components/App.tsx (hunk 31 above)
Comment (estherkim) [reply release-calendar/src/client/components/App.tsx:49]: Yes please, let's keep it consistent.

File: release-calendar/src/client/components/Calendar.tsx [hunk 34]
```diff
@@ -38,6 +38,10 @@ export class Calendar extends React.Component<CalendarProps, {}> {
           }}
           plugins={[dayGridPlugin, timeGridPlugin]}
           eventSources={this.props.events}
+          contentHeight={480} //will be 430 when header is added
```
Comment (danielrozenberg) [inline release-calendar/src/client/components/Calendar.tsx:41]: Can you make this a top-level const in this file? (e.g., `const CALENDAR_CONTENT_HEIGHT = 480;`)
Generally speaking if a value is not one of: `0`, `1`, `2`, `true`, `false`, `null`, `undefined`, or `''`, consider making it a const. If you end up reusing this value for a different component it makes it easier to change all the values at one :)

File: release-calendar/src/client/stylesheets/app.scss [hunk 35]
```diff
@@ -0,0 +1,66 @@
...
+
+.col-calendar {
+    @include container-shadow;
+    width: 63%;
+    float:right;
+}
+
+@media only screen and (max-width: 940px) {
+    .col-channel-table {
+        float:left;
```
Comment (danielrozenberg) [inline release-calendar/src/client/stylesheets/app.scss:31]: not: space after colon

File: release-calendar/src/client/stylesheets/app.scss [hunk 36]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+@media only screen and (max-width: 600px) {
+    .col-channel-table {
+        width: 70%;
+    }
+}
+
+body {
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
```
Comment (danielrozenberg) [inline release-calendar/src/client/stylesheets/app.scss:49]: I think it's weird to use OS-specific fonts; also, BlinkMacSystemFont and Segoe UI are not web-safe fonts and you're not importing them anywhere with @font-face, so this will look very different depending on the user's system. I'm far from an expert on web fonts though so if you need help figuring this out it's worth asking in the team group chat for help!

File: release-calendar/src/client/stylesheets/app.scss [hunk 37]
```diff
@@ -0,0 +1,66 @@
...
+    font-size: 14px;
+    line-height: 3em;
+    color: #24292e;
+    margin: 0px;
+}
+
+button {
+    -webkit-appearance: none;
```
Comment (danielrozenberg) [inline release-calendar/src/client/stylesheets/app.scss:57]: How would that look on browsers that aren't webkit? (like Firefox)

File: release-calendar/src/client/stylesheets/channelTable.scss [hunk 38]
```diff
@@ -0,0 +1,116 @@
...
+
+//Colors
+$lts: black;
+$stable: grey;
+$perc-beta: blue;
+$perc-experimental: green;
+$opt-in-beta: purple;
+$opt-in-experimental: silver;
+$nightly: red;
+$grey-border:rgb(216, 216, 216)
```
Comment (danielrozenberg) [inline release-calendar/src/client/stylesheets/channelTable.scss:42]: space 😅

File: release-calendar/src/client/components/Calendar.tsx (hunk 34 above)
Comment (ajwhatson) [reply release-calendar/src/client/components/Calendar.tsx:41]: oh, sure!

File: release-calendar/src/client/stylesheets/app.scss (hunk 36 above)
Comment (ajwhatson) [reply release-calendar/src/client/stylesheets/app.scss:49]: haha, I didn't make the connection that `apple-system` was "Apple". oops. I stole this font from an amp page. I'll ping someone about it for going about picking a better option.
